modules/ROOT/pages/index-policies.adoc

```diff
@@ -1,8 +1,25 @@
-= Security Policies
+= Edge Security Policies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
 
-Anypoint Security features a high-performance, reliable, and scalable service to enforce policies that apply to all nodes deployed to your Runtime Fabric. +
+Anypoint Security features a high-performance, reliable, and scalable service to enforce security policies that apply to all nodes deployed to your Runtime Fabric. 
 
-You can protect your APIs using these policies:
+== Prerequisites
+
+To configure and use the security policies, you must:
+
+. Have these permissions: 
```

Comment:
It's odd to go from one introductory fragment that ends with a colon to another.
Suggest not using the bullet list for only two items, considering this:

To configure and use the security policies, you must:
    1. Have API Manager Manage Policies permission and tokenization Manage Runtime Fabrics permission.
       See....
    2. Install...
        Anypoint....
    3. Enable inbound traffic....